modules/manage/partials/authentication.adoc

```diff
@@ -0,0 +1,1661 @@
+
+* *HTTP Proxy*: Access to the Kafka API impersonates the user whose credentials were used to authenticate to HTTP Proxy, and the user is subject to authorization restrictions by Redpanda ACLs. To support this design, Redpanda passes the username/password in memory to a SASL-enabled Kafka client.
+* *Schema Registry*: Authorization is "all or nothing": if the user presents a valid user account, then they have full read/write access.
+
+To add users to the Redpanda credential store that HTTP basic authentication uses, create users with xref:reference:rpk/rpk-acl/rpk-acl-user-create.adoc[`rpk acl user create`].
+
+ifdef::env-kubernetes[]
+When you <<enable-authentication, enable authentication>> in the Redpanda Helm chart, the Schema Registry API and the HTTP Proxy API are configured with basic authentication by default.
+
+To enable basic authentication for the Admin API:
+
+[tabs]
+======
+Helm + Operator::
++
+--
+.`redpanda-cluster.yaml`
+[,yaml,lines=14]
+----
+apiVersion: cluster.redpanda.com/v1alpha1
+kind: Redpanda
+metadata:
+  name: redpanda
+spec:
+  chartRef: {}
+  clusterSpec:
+    auth:
+      sasl:
+        enabled: true
+      secretRef: redpanda-superusers
+    config:
+      cluster:
+        admin_api_require_auth: true
+----
+
+```bash
+kubectl apply -f redpanda-cluster.yaml --namespace <namespace>
+```
+
+--
+Helm::
++
+--
+[tabs]
+====
+--values::
++
+.`enable-sasl.yaml`
+[,yaml,lines=7]
+----
+auth:
+  sasl:
+    enabled: true
+  secretRef: redpanda-superusers
+config:
+  cluster:
+    admin_api_require_auth: true
+----
++
+```bash
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+  --values sasl-enable.yaml --reuse-values
+```
+
+--set::
++
+[,bash,lines=5]
+----
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+  --set auth.sasl.enabled=true \
+  --set auth.sasl.secretRef=redpanda-superusers \
+  --set config.cluster.admin_api_require_auth=true
+----
+
+====
+--
+======
+endif::[]
+
+ifndef::env-kubernetes[]
+To enable basic authentication for the Admin API:
+
+. <<create-superusers,create a SCRAM superuser>> so that you can use `rpk` to create ACLs.
++
+`rpk` supports only basic authentication for the Admin API.
+
+. Enable authentication for the Admin API:
++
+[,bash]
+----
+rpk cluster config set admin_api_require_auth true
+----
+
+. Enable basic authentication:
++
+[,bash]
+----
+rpk cluster config set http_authentication '["BASIC"]'
+----
+
+To enable basic authentication for the HTTP Proxy or Schema Registry, set `authentication_method` to `http_basic`. For example, in `redpanda.yaml`, enter:
+
+[,yaml]
+----
+pandaproxy:
+  pandaproxy_api:
+  - address: "localhost"
+    port: 8082
+    authentication_method: http_basic
+schema_registry:
+  schema_registry_api:
+    address: "localhost"
+    port: 8081
+    authentication_method: http_basic
```

Comment:
The nesting here doesn't make it clear that `authentication_method` is also required for OIDC.

Also, can we have the pretty highlight one the `authentication_method` lines?